### PR TITLE
Blog App - Add integration tests

### PR DESCRIPTION
**Pull request by GuilhermeGiachellin**
# What was done in this PR:  (Pair programming)

- Created tests for all views using Capybara.
- Followed the guidelines for the tests.

Test each file individually!

## List of tests

login_test_capybara

user_index_test_capybara

user_show_test_capybara

post_index_test_capybara

post_show_test_capybara

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@ Metrics/ClassLength:
   Max: 150
 Metrics/BlockLength:
   IgnoredMethods: ['describe']
-  Max: 30
```
Comment (GzimAsani) [inline .rubocop.yml:29]: - [x] Update your readme file and explain to the users how can they test your project

File: Gemfile [hunk 2]
```diff
@@ -56,11 +56,12 @@ group :development do
   gem 'spring'
 end
 
-group :test do
+group :development, :test do
```
Comment (GzimAsani) [inline Gemfile:59]: - [x] 37 out of 54 tests are failing when I checked them
Screenshot from 2021-11-17 22-37-09

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Hi @GuilhermeGiachellin and @ArthurGC                           
Good job so far! Your code looks fine, good job :+1:  :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight:
- Good PR description :clap: 

## Required Changes ♻️ 
- [ ] _Check inline comments for more information :arrow_double_down:_ 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by GzimAsani — COMMENTED** [review]
N/A

**Review by descholar-ceo — APPROVED** [review]
Hi @ArthurGC , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**